### ISSUE TITLE
DST

### ISSUE THREAD

**Issue by sdemills** [benjymous/MWM-for-Android#77]
Last night the UK came out of DST. This morning the phone (GS2) is displaying the correct time but the MW is still an hour ahead. What is more, the MW time is 4 mins out (to be clearer, it is in total 56 mins ahead of the phone time). Now I have noticed in the past that when the MW is disconnected and left to run on its own then its internal timekeeping is not very accurate. So my conclusion is that MWM is no longer sending the time to the MW. If I stop the service and start it again then the MW time is updated correctly.

**Comment by benjymous** [comment]
Yup, my watch didn't get the updated time either. This is related to issue #61 - it seems like automatic RTC updates fail when the phone is sleeping, but work fine when the phone is awake. 

Possibly just need to obtain a wakelock whilst updating the watch time? 

If you manually change the time on your phone, you should see the watch update too.

**Comment by sdemills** [comment]
OK. How often do RTC updates take place? IOW if I had waited a bit would it
have happened anyway?

**Comment by benjymous** [comment]
It doesn't do it on a schedule, just automatically on connection, and if the time on the phone changes (or at least should do) - I have considered adding a daily time sync, just to make sure it stays right.

**Comment by sdemills** [comment]
Perhaps another option? - how often to send a time sync

**Comment by sdemills** [comment]
I notice after the last update that even on service recycle the time now no longer corrects. I have gotten around this at this point by using Tasker to fire, on a schedule, an Intent: android.intent.action.TIME_SET and that has fixed the problem for me.

Ideally, MWM should have an Option to correct the time on a user-defined schedule. For example: I'm using hourly between 7:00-23:00.
